### PR TITLE
Bump ACD and Smart Proxy for ACD

### PR DESCRIPTION
**Pull request by sbernhard**

### CODE REVIEW

File: packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec [hunk 1]
```diff
@@ -127,6 +129,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/test
 
 %changelog
+* Mon Mar 15 2021 Bernhard Suttner,,, 0.6.0-1
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec:132]:
```suggestion
* Mon Mar 15 2021 Bernhard Suttner <suttner@atix.de> 0.6.0-1
```

**Review by evgeni — CHANGES_REQUESTED** [review]
can you please fix the changelog entry? it's good to go then.

File: rel-eng/tito.props [hunk 2]
```diff
@@ -757,6 +757,8 @@ whitelist = ansiblerole-foreman_scap_client
   rubygem-ruby-nmap
   rubygem-safe_yaml
   rubygem-sentry-raven
+  rubygem-smart_proxy_acd
+  rubygem-smart_proxy_acd_core
```
Comment (evgeni) [inline rel-eng/tito.props:761]: you also need to add these to foreman-plugins-2.4-rhel7

**Review by evgeni — CHANGES_REQUESTED** [review]
oh, and tito

**Comment by sbernhard** [comment]
I guess, I missed the flight for 2.4.0 :-(

**Comment by evgeni** [comment]
> I guess, I missed the flight for 2.4.0 :-(

Luckily, for you, plugins are released independently, so you only missed being there on day 0.

**Review by evgeni — APPROVED** [review]

**Pull request merged**